mira/dkg/askemo/askemosw.py

```diff
@@ -0,0 +1,166 @@
+from pathlib import Path
+from typing import List
+
+import click
+import pandas as pd
+from mira.dkg.askemo.api import Term, write, HERE
+
+header_row = 1
+row_count = 59
```

Comment:
maybe you can remove this so we can arbitrarily extend the google sheet and make improvements witout needing code changes?

Reply:
I'll keep `header_row` so we can have comments at the top. I removed the bottom part so now we shouldn't need `row_count` at least.

Reply:
Resolved in https://github.com/indralab/mira/pull/139/commits/72a04723ed4462b610e30a2a650fac03e7f55035